org.doctales.ant-contrib.json

```diff
@@ -0,0 +1,18 @@
+[
+  {
+    "name": "org.doctales.ant-contrib",
+    "description": "Utility plugin which provides the ant-contrib library.",
+    "keywords": ["Ant"],
+    "homepage": "https://github.com/doctales/org.doctales.ant-contrib/",
+    "vers": "1.0.beta3",
```

Comment:
That should not be valid semver. `1.0.0.beta3` is, but not `1.0.beta3` because the first three version tokens are numbers.

Reply:
Note that we use SemVer 2 and this has changed between 1 and 2. In SemVer 2 this should be `1.0.0-beta3` (notice the hyphen separator).

I also suggest rebasing the PR on `dita-ot/registry` `master`, to get the latest validation code.